pkg/util/priority/priority.go

```diff
@@ -77,6 +77,7 @@ func getDefaultPriorityClass(ctx context.Context, client client.Client) (*schedu
 	// we pick the one with the lowest priority value.
 	var defaultPC *schedulingv1.PriorityClass
 	for _, pci := range pcs.Items {
+		pci := pci
```

Comment:
This is a pretty common mistake we sometimes see. You can test why it is a problem with the "priorityClass is unspecified and multiple global defaults exist" test case

Reply:
I still can't understand this. Can you add more details or informations for it?

Reply:
The article below seems to explain the problem in detail. In short, the variable `pci` is reused for each iteration, so referring to the pointer of `pci` ends up pointing to the last element of `pcs.Items`🙂 

https://medium.com/swlh/use-pointer-of-for-range-loop-variable-in-go-3d3481f7ffc9

Reply:
`pci := pci` may be confusing so I renamed it